src/otx/algorithms/visual_prompting/tasks/inference.py

```diff
@@ -179,7 +179,7 @@ def get_model(config: DictConfig, train_type: TrainType, state_dict: Optional[Or
                         SegmentAnything as VisualPrompter,
                     )
                 elif train_type == TrainType.Zeroshot:
-                    from otx.algorithms.visual_prompting.adapters.pytorch_lightning.models import (
+                    from otx.algorithms.visual_prompting.adapters.pytorch_lightning.models import (  # type: ignore[assignment] # noqa: E501
```

Comment:
Could you explain why we need this noqa?

Reply:
That line becomes longer than 120 due to `# type: ignore[assignment]`.

Reply:
I think mypy was upset because assigning `VisualPrompter` was written twice, even though if-statement was used.

Reply:
```python
src/otx/algorithms/visual_prompting/tasks/inference.py:182: error: Incompatible import of "VisualPrompter" (imported name has type "Type[ZeroShotSegmentAnything]", local name has type "Type[SegmentAnything]")  [assignment]
```